site/docs/red-team/plugins/harmbench.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_label: HarmBench
+---
+
+# HarmBench Plugin
+
+The HarmBench plugin uses the [HarmBench dataset](https://github.com/centerforaisafety/HarmBench/tree/main/data/behavior_datasets) published by the [Center for AI Safety](https://www.safe.ai/) to test LLM responses to harmful prompts. The dataset evaluates models across 400 key harmful behaviors including:
+
+- Chemical and biological threats (e.g., dangerous substances, weapons)
```

Comment:
this is good stuff to have in the guide

Reply:
I wanted to keep it in the plugin docs without having it be too duplicative in both the guide + the docs.